bugsnag-plugin-android-ndk/src/main/jni/bugsnag.c

```diff
@@ -132,6 +132,7 @@ void bugsnag_notify_env(JNIEnv *env, const char *name, const char *message,
   jbyteArray jmessage = NULL;
 
   bugsnag_stackframe stacktrace[BUGSNAG_FRAMES_MAX];
+  memset(stacktrace, 0, sizeof(bugsnag_stackframe) * BUGSNAG_FRAMES_MAX);
```

Comment:
Its an array, and declared immediately beforehand (instead of being passed as a parameter to this function, so you can grab the size directly:
```suggestion
  memset(stacktrace, 0, sizeof(stacktrace));
```